tests/test_cli_client.py

```
@@ -515,6 +517,123 @@ def handler(request):
         )
 
 
+class TestJJIClientIssueTokens:
+    def test_preview_github_issue_with_tokens(self):
+        def handler(request):
+            body = json.loads(request.content)
+            assert body["github_token"] == "ghp_test123"
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to: "github_token"

(S105)